datalake/backend/tests/collection.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "test_dataset",
+    "stac_version": "1.0.0-beta.2",
+    "description": "This is dataset description.",
+    "links": [
+        {
+            "rel": "item",
+            "href": "s3://s3-bucket-name/datasets/xyz/1234.json",
+            "type": "application/json"
+        },
+        {
+            "rel": "root",
+            "href": "s3://s3-bucket-name/catalog.json",
+            "type": "application/json"
+        },
+        {
+            "rel": "parent",
+            "href": "s3://s3-bucket-name/datasets/xyz/catalog.json",
+            "type": "application/json"
+        },
+        {
+            "rel": "self",
+            "href": "s3://s3-bucket-name/datasets/xyz/collection.json",
+            "type": "application/json"
+        }
+    ],
+    "title": "Dataset Title",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    172.05956425817075,
+                    -40.91288934049462,
+                    172.51785842526388,
+                    -40.58580513954452
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2003-01-01T00:00:00Z",
+                    "2003-12-31T00:00:00Z"
+                ]
+            ]
+        }
+    },
+    "license": "CC-BY-4.0",
+    "stac_extensions": [
+        "eo",
+        "proj",
+        "linz"
+    ],
+    "keywords": [
```

Comment:
We should make sure this dataset is as simple as humanly possible, to make sure the test and all its components are maintainable. Several of the attributes in this file, such as `keywords`, are not needed to create a valid STAC file. See for example `MINIMAL_VALID_STAC_OBJECT`.

Reply:
In 7c9a88f4c503a3bb403dab48a0d58a217af934f3 I used `MINIMAL_VALID_STAC_OBJECT` . I leave it on you to extend it when it will be needed.